openforcefield/tests/test_molecule.py

```diff
@@ -208,6 +208,7 @@ def mini_drug_bank(xfail_mols=None, wip_mols=None):
 
 # Missing stereo in OE but not RDK:  'DrugBank_2987', 'DrugBank_3502', 'DrugBank_4161',
 # 'DrugBank_4162', 'DrugBank_6531', 'DrugBank_1700',
+drugbank_stereogenic_in_rdkit_but_not_openeye = ['DrugBank_5329']
```

Comment:
I notice the similar variables above and below are sets, not lists. Probably not consequential but I like the uniformity

Reply:
Good find. Resolved in 9d2e373